go.mod

```diff
@@ -33,3 +33,5 @@ require (
 	github.com/zclconf/go-cty-debug v0.0.0-20191215020915-b22d67c1ba0b
 	golang.org/x/tools v0.0.0-20200323144430-8dcfad9e016e
 )
+
+replace github.com/hashicorp/terraform-schema v0.0.0-20210601142728-f0b01c296a30 => github.com/aztfmod/terraform-schema v0.0.0-20210604102211-dbbfe1505190
```

Comment:
I'll update once https://github.com/hashicorp/terraform-schema/pull/53 is merged